src/LiveComponent/src/Attribute/AsLiveComponent.php

```diff
@@ -21,6 +21,32 @@
 #[\Attribute(\Attribute::TARGET_CLASS)]
 final class AsLiveComponent extends AsTwigComponent
 {
+    private string $defaultAction;
+
+    public function __construct(string $name, ?string $template = null, string $defaultAction = '__invoke')
```

Comment:
Yeah, still new to attributes but I thought (since most Symfony users are converting annotations to attributes) the common way to configure would be with named parameters.

Reply:
For ones with defaults, I guess just keep tacking onto the end to keep BC. Hopefully there will be no more required arguments....